spec/support/matchers/have_form_field.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
```

Comment:
Can we just release `hyrax-spec` `0.3.1`, rather than copying this in?

Reply:
@no-reply sure, i just didn't want to overlook it.